### ISSUE TITLE
displayModeBar = false

### ISSUE THREAD

**Issue by akubisch** [sglyon/PlotlyJS.jl#185]
Hi,
I am happily using this great package! Yet, I was just wondering, how it is possible to hide the mode bar using plotlyJS.jl!? I know it is possible (see [here](https://plot.ly/javascript/configuration-options/#hide-the-modebar)). But what is the correct position for this argument?

Thanks a lot in advance!

**Comment by sglyon** [comment]
Hi @akubisch thanks for reaching out -- glad you are liking the package!

I actually haven't hooked this up yet, but would be very happy to accept a pull request that implements this. 

I could provide some pointers on where/how to get started. Would you be interested in contributing a pull request with this feature?

See also #108

**Comment by akubisch** [comment]
Hi @sglyon,

Thanks for the swift response! I would surely be interested in contributing a pull request, it just depends a bit on the potential load of work associated with it ;-) If you could give me some hints I could have a look at it.

Also - before opening a new issue: is there an easy way to change the locale of date axes?

**Comment by sglyon** [comment]
For locale, I think you can set `xaxis_tickformat` using this mini language: https://github.com/d3/d3-time-format/blob/master/README.md#locale_format

- - -


I think one way we could implement this in a few steps:

**1** Add an additional field on the [`SyncPlot` type](https://github.com/sglyon/PlotlyJS.jl/blob/4f0c99aed8a5993a08041d02cf4cd631908c4dbb/src/display.jl#L61-L64) so that it looks like this:

```julia
struct SyncPlot{TD<:AbstractPlotlyDisplay}
    plot::Plot
    view::TD
    config::PlotlyBase.PlotlyAttribute
end
```

**2**

Have the [`plot` method](https://github.com/sglyon/PlotlyJS.jl/blob/4f0c99aed8a5993a08041d02cf4cd631908c4dbb/src/display.jl#L66) look like this:

```julia
function plot(args...; config::PlotlyBase.PlotlyAttribute=attr(showLink=false), kwargs...)
    SyncPlot(Plot(args...; kwargs...), config)
end
```

**3**  

Have the [`SyncPlot` method](https://github.com/sglyon/PlotlyJS.jl/blob/4f0c99aed8a5993a08041d02cf4cd631908c4dbb/src/display.jl#L133) look like this:

```julia
function SyncPlot(p::Plot, config::PlotlyBase.PlotlyAttribute=attr())
    SyncPlot(p, DEFAULT_DISPLAY[](p), config)
end
```

**4** Find all places where we call the javascript function `Plotly.newPlot` and make sure to pass the `config` settings from the SyncPlot

This includes

1. [`script_content` here](https://github.com/sglyon/PlotlyJS.jl/blob/4f0c99aed8a5993a08041d02cf4cd631908c4dbb/src/display.jl#L16) -- note we might need to add a `config` argument here because this takes in a Plot instead of a SyncPlot
2. [`display_blink` here](https://github.com/sglyon/PlotlyJS.jl/blob/4f0c99aed8a5993a08041d02cf4cd631908c4dbb/src/displays/electron.jl#L115) -- note `ElectronPlot` is just a sync plot that has an ElectronDisplay, so the new `config` field we created in step 1 will be available. Also, there are 2 usages of `Plotly.newPlot` in that function
3. [`html_body` here](https://github.com/sglyon/PlotlyJS.jl/blob/4f0c99aed8a5993a08041d02cf4cd631908c4dbb/src/displays/ijulia.jl#L26)

- - -

When we are doing step 4 from above, I think we might want to make sure that `showLink` is set to false whenever it isn't explicitly passed in `config` so that the new feature is backwards compatible.

Hopefully that makes some sense. If not, please tell me!

**Comment by akubisch** [comment]
It very much does and I think I should be able to handle that :-) I will try to take care of it over the weekend, but cannot promise it'll work out.

**Comment by sglyon** [comment]
included in #208

**Comment by sglyon** [comment]
For now, please use the master branch

```julia
] # enter pkg mode
(v1.0) pkg> checkout PlotlyJS
```

If you want save figures you can do

```
] # enter package mode
(v1.0) pkg> add https://github.com/sglyon/ORCA.jl.git#master
```

If this doesn't resolve your issue here, please reopen the issue and leave a comment describing what went wrong
